presto-docs/src/main/sphinx/release/release-0.223.rst

```diff
@@ -0,0 +1,23 @@
+=============
+Release 0.223
+=============
+
+General Changes
+---------------
+
+* Fix a bug where a connector may not optimize query plan if the corresponding
+  plugin is installed after the server is started.
+* Add support for multibyte characters in :func:`date_format`.
+* Add support for using long polling for task information updates to improve
```

Comment:
As per @tdcmeehan 's suggestion we should change this to talk about the impact.

```
Improve coordinator CPU utilization by introducing an option to use long polling for task information update.
```

And since the ordering of the notes are:

1) Fix
2) Improve
3) Add

we should point it to the 2nd place.